woof/target-shield.html

```diff
@@ -0,0 +1,61 @@
+<link href="./projects.css" rel="stylesheet">
+<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+<h1><a href="./index.html"><span class="glyphicon glyphicon-arrow-left"></span></a> Target Shield</h1>
+<div class="container">
+	<div class="block">
+		<p>Protect your target from infiltrators by blocking them before they reach base. Use your shield as a last line of defense. Play this game on any device, including mobile! </p>
+		<br><a target="_blank" href="./code/block-dodger-code.html">Play this demo in full screen.</a>
+	</div>
+	<div class="block">
+		<p>1) Set your backdrop and create your target.</p>
+		<br><a target="_blank" href="./snippets/target-shield-step1.html">Play this demo in full screen.</a>
+	</div>
+	<div class="block">
+		<p>2) Create your blocker sprite and make it follow your mouse. </p>
+		<br><a target="_blank" href="./snippets/target-shield-step2.html">Play this demo in full screen.</a>
+	</div>
+	<div class="block">
+		<p>3) Create your shield and have it only appear when you press a key. ("S" is used in the demo.)  </p>
+		<br><a target="_blank" href="./snippets/target-shield-step3.html">Play this demo in full screen.</a>
+	</div>
+	<div class="block">
+		<p>4) Make the shield disappear after a few seconds and only allow the player to deploy it once. Tell the player that it is for one-time use only. </p>
```

Comment:
Maybe you should have the text that says you can only use the shield once update after you use it to reflect the fact that you can no longer use it?